### ISSUE TITLE
argparse.ArgumentError: argument --host: conflicting option string: --host

### ISSUE THREAD

**Issue by POD666** [pytest-dev/pytest-selenium#215]
Hi, 
here is the issue I've faced:
```
$ python3 -V
Python 3.7.0
$ python3 -m venv testenv
$ source testenv/bin/activate 
$ pip install pytest-selenium
...
Successfully installed atomicwrites-1.3.0 attrs-18.2.0 certifi-2018.11.29 chardet-3.0.4 idna-2.8 more-itertools-5.0.0 pluggy-0.8.1 py-1.7.0 pytest-4.2.0 pytest-base-url-1.4.1 pytest-html-1.20.0 pytest-metadata-1.8.0 pytest-selenium-1.15.1 pytest-variables-1.7.1 requests-2.21.0 selenium-3.141.0 six-1.12.0 urllib3-1.24.1
$ pytest
Traceback (most recent call last):
...
File ".../argparse.py", line 1529, in _handle_conflict_error
    raise ArgumentError(action, message % conflict_string)
argparse.ArgumentError: argument --host: conflicting option string: --host
```
Any suggestions?

**Comment by BeyondEvil** [comment]
Any news on this @POD666 ?

**Comment by MidCheck** [comment]
Hello, I have encountered the same error:
```
>>> import argparse
>>> parser = argparse.ArgumentParser()
>>> parser.add_argument('-h', '--host')
Traceback (most recent call last):
...
File "/usr/lib64/python3.7/argparse.py", line 1529, in _handle_conflict_error
    raise ArgumentError(action, message % conflict_string)
argparse.ArgumentError: argument -h/--host: conflicting option string: -h
```
Finally, I think the cause of the error might be that the function add_argument() specifies the '--host' parameter, which may conflict with the default '-h' (help) parameter, see: https://docs.python.org/zh -cn/3.7/library/argparse.html

**Comment by BeyondEvil** [comment]
@MidCheck 

Can you please share information regarding your environment?

OS
python version
pytest and plugin versions
anything else that you might think is pertinent to reproduce the issue.

Also, a minimal test case that reproduces the issue would help.

Thanks!

**Comment by BeyondEvil** [comment]
I'm going to close this issue due to lack of interaction. If you're still seeing this and can successfully reproduce it - feel free to reopen.

**Comment by topher-brennan** [comment]
I had this issue. I ran into it while trying to use django-pdb (specifically by putting `'django_pdb'` in `INSTALLED_APPS` and `'django_pdb.middleware.PdbMiddleware'` in `MIDDLEWARE` both in my settings.py file). Wound up just removing it because I decided I didn't really need them for testing and the problem disappeared.

**Comment by Dude-X** [comment]
I am thinking that the name `--host` is too generic and maybe we should use something a bit wordy like `-grid-host` or some other way to "namespace" the option to prevent clashing with other plugins and libraries.

**Comment by BeyondEvil** [comment]
> I am thinking that the name `--host` is too generic and maybe we should use something a bit wordy like `-grid-host` or some other way to "namespace" the option to prevent clashing with other plugins and libraries.

100% agree!

We'll deprecate `--host` and `--port` in favor of `--grid-host` and `--grid-port`. I'll draw up a PR.
